### PR TITLE
chore: prepend @aws-amplify pt.2

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes
Prepend @aws-amplify to dependencies so malicious users may not impersonate Amplify on npm

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by jhockett — APPROVED** [review]

File: packages/amplify-cli/API.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 import { $TSAny } from 'amplify-cli-core';
 import { CommandLineInput } from 'amplify-cli-core';
-import { IAmplifyLogger } from 'amplify-cli-logger/lib/IAmplifyLogger';
+import { IAmplifyLogger } from '@aws-amplify/amplify-cli-logger/lib/IAmplifyLogger';
```
Comment (sobolk) [inline packages/amplify-cli/API.md:11]: this import doesn't look good. we shouldn't import from `lib` .

File: packages/amplify-cli/src/domain/amplify-usageData/FlowReport.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 /* eslint-disable spellcheck/spell-checker */
+import { getAmplifyLogger, Redactor } from '@aws-amplify/amplify-cli-logger';
+import { IAmplifyLogger } from '@aws-amplify/amplify-cli-logger/lib/IAmplifyLogger';
```
Comment (sobolk) [inline packages/amplify-cli/src/domain/amplify-usageData/FlowReport.ts:3]: same here.

**Review by goldbez — DISMISSED** [review]

File: packages/amplify-cli/src/domain/amplify-usageData/FlowReport.ts (hunk 2 above)
Comment (evcodes) [reply packages/amplify-cli/src/domain/amplify-usageData/FlowReport.ts:3]: fixing this and running verify-api-extract:clean should fix both

File: packages/amplify-cli-logger/API.md [hunk 3]
```diff
@@ -1,14 +1,28 @@
...
 
+// @public (undocumented)
+export interface IAmplifyLogger {
+    // Warning: (ae-forgotten-export) The symbol "LogErrorPayload" needs to be exported by the entry point index.d.ts
+    //
+    // (undocumented)
+    logError(content: LogErrorPayload): void;
+    // (undocumented)
+    loggerEnd(): void;
+    // Warning: (ae-forgotten-export) The symbol "LogPayload" needs to be exported by the entry point index.d.ts
```
Comment (sobolk) [inline packages/amplify-cli-logger/API.md:18]: we should also export `LogErrorPayload` and `LogPayload`

**Review by sobolk — APPROVED** [review]

**Review by jhockett — APPROVED** [review]

**Pull request merged**